autodistill/core/base_model.py

```diff
@@ -4,7 +4,7 @@
 
 import supervision as sv
 
-from autodistill.core import Ontology
+from autodistill.core.ontology import Ontology
```

Comment:
Is this a purposeful change? Was it broken before?

Reply:
Yes. Imports are causing a few issues, like this one: https://github.com/autodistill/autodistill/issues/30